### PR TITLE
Added unstable potions to vendorProfResults issue

### PR DESCRIPTION
**Pull request by Evpok**
Solves https://github.com/Phr33d0m/NW-Profession-Bot/issues/169

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -5371,7 +5371,7 @@ function addProfile(profession, profile, base){
...
+                pattern: /^Crafted_(Jewelcrafting_Waist_Offense_3|Jewelcrafting_Neck_Defense_3|Jewelcrafting_Waist_Defense_3|Med_Armorsmithing_T3_Chain_Armor_Set_1|Med_Armorsmithing_T3_Chain_Pants2|Med_Armorsmithing_T3_Chain_Shirt2|Med_Armorsmithing_T3_Chain_Helm_Set_1|Med_Armorsmithing_T3_Chain_Pants|Med_Armorsmithing_T3_Chain_Boots_Set_1|Hvy_Armorsmithing_T3_Plate_Armor_Set_1|Hvy_Armorsmithing_T3_Plate_Pants2|Hvy_Armorsmithing_T3_Plate_Shirt2|Hvy_Armorsmithing_T3_Plate_Helm_Set_1|Hvy_Armorsmithing_T3_Plate_Boots_Set_1|Leatherworking_T3_Leather_Armor_Set_1|Leatherworking_T3_Leather_Pants2|Leatherworking_T3_Leather_Shirt2|Leatherworking_T3_Leather_Helm_Set_1|Leatherworking_T3_Leather_Boots_Set_1|Tailoring_T3_Cloth_Armor_Set_3|Tailoring_T3_Cloth_Armor_Set_2|Tailoring_T3_Cloth_Armor_Set_1|Tailoring_T3_Cloth_Pants2_Set2|Tailoring_T3_Cloth_Shirt2|Tailoring_T3_Cloth_Helm_Set_1|Artificing_T3_Pactblade_Temptation_5|Artificing_T3_Icon_Virtuous_5|Weaponsmithing_T3_Dagger_4)|^Potion_Unstable(_[1-5])*$/,
```
Comment (Phr33d0m) [inline main.js]: One small note, change `(_[1-5])*$` to `_([1-5])*$`.

Comment (Evpok) [inline main.js]: IIRC the minor unstable potion is just `Potion_Unstable` but as I sold them I can't check from my inventory :p

Comment (Evpok) [inline main.js]: After checking, it appears it isn't. Changing now. For future reference, I checked using `window.client.dataModel.model.craftinglist.craft_Alchemy.entries.filter(function(entry){if(entry.def){return entry.def.name.match(/Experiment/);}return false}).map(function(entry){return entry.rewards[0].hdef}).filter(function(entry){if(entry.match(/Unstable/)){return entry}})`

**Comment by noonereally** [comment]
Thanks for the PR

**Pull request merged**